### PR TITLE
making cast count variable

### PR DESCRIPTION
**Pull request by yhedholm**
The biggest change was adding a castCount field in the Cast object.

### CODE REVIEW

File: frontend/rolecall/src/app/api/cast_api.service.ts [hunk 1]
```diff
@@ -108,28 +109,24 @@ export class CastApi {
...
         return {
           data: {
-            casts: val.data.map(rawCast => {
-              let groups: {
-                position_uuid: string,
-                group_index: number,
-                members: { uuid: string, position_number: number }[]
-              }[] = [];
+            casts: result.data.map(rawCast => {
+              let higestCastNumber = 0;
```
Comment (mikzat) [inline frontend/rolecall/src/app/api/cast_api.service.ts:128]: spelling: "highest"

File: frontend/rolecall/src/constants.ts [hunk 2]
```diff
@@ -44,3 +44,6 @@ export const COLORS = {
   offoffWhite: '#d9d9d9',
   AAOrange: '#ffae00'
 };
+
+/** For Casting Screen. */
```
Comment (mikzat) [inline frontend/rolecall/src/constants.ts:48]: Let's also describe what it means.

File: frontend/rolecall/src/app/api/cast_api.service.ts (hunk 1 above)
Comment (yhedholm) [reply frontend/rolecall/src/app/api/cast_api.service.ts]: Fixed.

File: frontend/rolecall/src/constants.ts (hunk 2 above)
Comment (yhedholm) [reply frontend/rolecall/src/constants.ts]: Done.

**Review by mikzat — APPROVED** [review]

**Pull request merged**